{
	"schema_version": "1.6.0",
	"id": "RHSA-2022:6766",
	"modified": "2024-09-13T22:15:07Z",
	"published": "2024-09-13T22:15:07Z",
	"related": [
		"CVE-2015-20107",
		"CVE-2020-10735",
		"CVE-2021-28861"
	],
	"summary": "Red Hat Security Advisory: rh-python38-python security update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:H/A:L"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python",
				"purl": "pkg:rpm/redhat/rh-python38-python"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python-debug",
				"purl": "pkg:rpm/redhat/rh-python38-python-debug"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python-debuginfo",
				"purl": "pkg:rpm/redhat/rh-python38-python-debuginfo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python-devel",
				"purl": "pkg:rpm/redhat/rh-python38-python-devel"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python-idle",
				"purl": "pkg:rpm/redhat/rh-python38-python-idle"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python-libs",
				"purl": "pkg:rpm/redhat/rh-python38-python-libs"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python-rpm-macros",
				"purl": "pkg:rpm/redhat/rh-python38-python-rpm-macros"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python-srpm-macros",
				"purl": "pkg:rpm/redhat/rh-python38-python-srpm-macros"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python-test",
				"purl": "pkg:rpm/redhat/rh-python38-python-test"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_software_collections:3::el7",
				"name": "rh-python38-python-tkinter",
				"purl": "pkg:rpm/redhat/rh-python38-python-tkinter"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:3.8.14-1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6766.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2022:6766"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#moderate"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1834423"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2075390"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2120642"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2022/rhsa-2022_6766.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2015-20107"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2015-20107"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2015-20107"
		},
		{
			"type": "ARTICLE",
			"url": "https://python-security.readthedocs.io/vuln/mailcap-shell-injection.html"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2020-10735"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2020-10735"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-10735"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2021-28861"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2021-28861"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2021-28861"
		}
	]
}
